macosx/QuickLookPlugin/pt.lproj/Localizable.strings

```diff
@@ -1,14 +1,14 @@
 /* quicklook file header */
```

Comment:
The folder `QuickLookPlugin/pt.lproj` should be deleted: we only support "pt_PT", not "pt".